core/src/main/java/com/opensymphony/xwork2/ognl/accessor/CompoundRootAccessor.java

```diff
@@ -217,6 +217,7 @@ public Object callMethod(Map context, Object target, String name, Object[] objec
             return null;
         }
 
+        Throwable reason = null;
```

Comment:
(Oops ... forgot to include this comment in the review).

Assuming the loop logic will continue to process more than 1 iteration, I suggest a small optimization:
 - Move Line **227**:  `Class[] argTypes = getArgTypes(objects);`  -- to Line **219/220** (outside of the loop).
_Should_ be safe as the argument types don't change across the loop iterations.

Reply:
👍 fixed